packages/cnpj-fmt/src/cnpj-fmt.php

```
@@ -2,35 +2,71 @@
         $hiddenEnd,
         $dotKey,
         $slashKey,
         $dashKey,
+        $escape,
+        $encode,
         $onFail,
     );
 
-    return $formatter->format($cnpjString);
+    return $formatter->format($cnpjInput);
 }
```

PHPMD (2.15.0)

[warning] The function cnpj_fmt has 12 parameters. Consider reducing the number of parameters to less than 10. (undefined)

(ExcessiveParameterList)